Header header
float32 enc_count

